.github/workflows/pythonpackage.yml

```diff
@@ -29,6 +30,10 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Set up NeuroDebian
+      run: bash <(wget -q -O- http://neuro.debian.net/_files/neurodebian-travis.sh)
+    - name: Install FSL
+      run: sudo apt-get install -y fsl
```

Comment:
do you really want to install fsl from neurodebian? it hasn't updated fsl since 5.0.9. perhaps in this case we can run using neurodocker, or add a testkraken based evaluation to this against versions of fsl?

Reply:
Fine switching to Neurodocker; this is just what I used in pydra-nipype1 to be able to run some tests. Along the lines of my previous comment, I now think I'd rather isolate these extra installs.

IDK enough about using testkraken here, but if it's ready to do that, I'm happy to try to build the CI template around it.

Reply:
let me ping @djarecka here. we would like to test the pydra-fsl tasks within containers of versions of FSL. it seems including a testkraken spec would be good. but perhaps you can suggest how it should/could be done?

Reply:
@satra - but we would like to run all tests from this repo?

Reply:
to start with yes.

Reply:
and ideally one should be able to run the tests both locally and through github actions.

Reply:
ok, let me see how testkraken would fit here (was mostly testing on a specific structure)